presto-iceberg/src/main/java/com/facebook/presto/iceberg/ManifestFileCache.java

```diff
@@ -71,4 +71,9 @@ public boolean isEnabled()
     {
         return enabled;
     }
+
+    public void invalidateManifestFileCache()
```

Comment:
is this method even necessary? Can't you just call `invalidateAll` using the object directly since it extends from `Cache<>`?

Reply:
I thought of pushing the invalidation method logic to the cache method in case we plan to extend this more. Let me know if you still think it should still be in procedure call itself?

Reply:
There isn't really any logic to this method currently. I think we should just call `invalidateAll` directly in the procedure